src/ui/inspector/StylesInspectorView.vue

```diff
@@ -24,16 +25,38 @@ export default {
         this.openmct.selection.off('change', this.updateSelection);
     },
     methods: {
+        getStyleProperties(item) {
+            let styleProps = {};
+            for (let key in item) {
+                if (item.hasOwnProperty(key)) {
+                    let styleProp = getStyleProp(key, item[key]);
+                    if (styleProp) {
+                        styleProps = {
```

Comment:
Seems like a lot of unnecessary object instantiation here. Could we use `Object.assign()` here instead?

Reply:
Done. Also changed other occurrences of object deconstruction to Object.assign.